src/collections/Customers.ts

```diff
@@ -54,6 +65,7 @@ export const Customers: CollectionConfig = {
                   relationTo: 'blocks',
                   required: true,
                   filterOptions: ({ data }) => {
+                    if (!data.area) return true
```

Comment:
This change would cause inconsistent data in customers. eg: if we select block first for example `Block A` from Naya Nazimabad and then select area `Bahria Town` this change allow this and make data inconsistent.